### PR TITLE
Created a C++ gitignore

### LINKED ISSUE #105: Add binary files to .gitignore
### Description
Many of us are pushing binary files by mistake, gitignore can help in not committing binary files.
<Remove this line> The title should be short and relevant.

### PR DESCRIPTION
**Pull request by dhruvrkashyap**
Closes #105 
<Do NOT delete this template>
Description: Add a .gitignore

#### Checklist
<Put an x between square brackets wherever appropriate without any spaces like [x]>

- [ ] I've followed the Contributing guidelines provided in the repository.
- [ ] I've made the changes which were demanded in the linked issue.
- [ ] My code gave a clean output on debugging. (NO warnings/errors)

### CODE REVIEW

**Comment by anktrj** [comment]
You need to edit the .gintignore file. Add these lines to that file.
Also, the binary files have no extensions. We have to find a way to ignore those files only

**Comment by sanskarseth** [comment]
https://stackoverflow.com/questions/5711120/gitignore-without-binary-files#:~:text=Up%20vote%200-,The%20.,binary%20file%20names%20to%20your%20. refer this and do the changes in .gitignore file

**Comment by anktrj** [comment]
any updates @dhruvrkashyap

**Comment by dhruvrkashyap** [comment]
I have added something, can you check if this is what you are looking for?
Reference: https://stackoverflow.com/questions/19023550/how-do-i-add-files-without-dots-in-them-all-extension-less-files-to-the-gitign/19023985#19023985

File: C++.gitignore [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# object files
```
Comment (anktrj) [inline C++.gitignore:1]: add all these lines to the existing .gitignore file and you're good to go!

**Comment by dhruvrkashyap** [comment]
i have updated .gitignore.

**Pull request merged**